run_markdownlint.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# Runner script to ensure
+# - independent from working directory
+
+# Change working directory to project root
+SOURCE=${BASH_SOURCE[0]}
+SCRIPT_DIR="$(dirname "$SOURCE")"
+PROJECT_ROOT="$SCRIPT_DIR"
+cd "$PROJECT_ROOT" || { echo "Could not change directory"; exit 1; }
+
+# Run markdownlint-cli2 (in a Docker container for convenience)
+docker run -v $PWD:/workdir davidanson/markdownlint-cli2:v0.12.1 --config .markdownlint-cli2.yaml "#node_modules"
```

Comment:
You are right on a real Pi this could be an issue.

You can also install markdownlint-cli2 with npm (see https://github.com/DavidAnson/markdownlint-cli2?tab=readme-ov-file#install). So far I didn’t bother to test this, because docker was a one command solution.

will check it out.

Reply:
done

Reply:
The new way implies that Node has been installed for the webapp?

Though i can be opt-out as the webapp is optional, i think this should be fine. 
Developer should have the full setup. Maybe we can make a path for this in the new installations procedure (#2249 ).

Reply:
Has anyone of you already had a look at https://pre-commit.com/ ?
Stumbled over it in the markdownlint docs. Maybe this would make the pre-commit handling easier for us?

Reply:
> The new way implies that Node has been installed for the webapp?
> 
> Though i can be opt-out as the webapp is optional, i think this should be fine. 
> Developer should have the full setup. Maybe we can make a path for this in the new installations procedure (#2249 ).

Yeah, this was what I was thinking, too.

I think we can expect that a developer has usually Node installed and it's still less intrusive on a Pi than docker.
And you can always disable the pre commit hook.

Reply:
> Has anyone of you already had a look at https://pre-commit.com/ ?

I have seen that, too and I also wondered if this could help us 😀